### PR TITLE
Fix #6791 : Table styling on Settings page made consistent to that on entity page

### LINKED ISSUE #6791: Table styling on Settings page is different from the styling on entity page.

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on the Entity Table component:
- Replaced react-table Table component with antd Table
- Fixed Unit tests for above changes.
- Fixed failing cypress tests.

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2022-08-27 at 7 47 35 PM
Screenshot 2022-08-27 at 7 48 05 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@aniketkatkar97, Can we add the screenshots?

**Comment by Sachin-chaurasiya** [comment]
Can we rename all `EntityTableV1` to `EntityTable`?

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.test.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.test.tsx:1]: Can we remove this?

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTableV1.component.tsx [hunk 2]
```diff
@@ -0,0 +1,925 @@
...
+    const fqn = cell?.fullyQualifiedName || '';
+    const columnName = getPartialNameFromTableFQN(fqn, [FqnPart.NestedColumn]);
+    // wrap it in quotes if dot is present
+
+    return columnName.includes(FQN_SEPARATOR_CHAR)
+      ? `"${columnName}"`
+      : columnName;
+  };
+
+  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTableV1.component.tsx:330]: We are using `antd` table, So, Can we use `antd` cell type here and in other places as well?

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.test.tsx (hunk 1 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTable.test.tsx]: removed

File: openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTableV1.component.tsx (hunk 2 above)
Comment (aniketkatkar97) [reply openmetadata-ui/src/main/resources/ui/src/components/EntityTable/EntityTableV1.component.tsx]: Changes made as suggested.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**